### PR TITLE
For packages, add necessary manifest fields

### PR DESCRIPTION
**Pull request by mikedotexe**
All the packages are on crates.io now, and in that process it makes us add useful things like a description.

Did the thing

### CODE REVIEW

File: contracts/mod-generic/Cargo.toml [hunk 1]
```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 authors = []
 edition = "2021"
 repository = "https://github.com/CronCats/cw-croncat"
+license = { workspace = true }
+description = "Allows for CosmWasm raw queries through this module, helpful when making CronCat tasks with queries"
```
Comment (Buckram123) [inline contracts/mod-generic/Cargo.toml:8]: This contract is actually to make queries to any contract and check any field of it. I guess something like "Allows to deserialize any contract's queue response and check the results of any fields"

File: packages/croncat-sdk-manager/Cargo.toml [hunk 2]
```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 authors = { workspace = true }
 edition = "2021"
 repository = "https://github.com/CronCats/cw-croncat"
+license = { workspace = true }
+description = "CronCat Manager types, messages, and errors."
```
Comment (Buckram123) [inline packages/croncat-sdk-manager/Cargo.toml:8]: I guess this one missing part about cosmwasm, but maybe I like this description even more

File: packages/croncat-sdk-tasks/Cargo.toml [hunk 3]
```diff
@@ -4,6 +4,8 @@ version = "0.1.0"
 authors = { workspace = true }
 edition = "2021"
 repository = "https://github.com/CronCats/cw-croncat"
+license = { workspace = true }
+description = "CronCat tasks package containing CosmWasm types and messages"
```
Comment (Buckram123) [inline packages/croncat-sdk-tasks/Cargo.toml:8]: Doesn't it have errors?

**Review by Buckram123 — APPROVED** [review]
Thanks! Don't know if we have to make standardized descriptions

File: contracts/mod-generic/Cargo.toml (hunk 1 above)
Comment (mikedotexe) [reply contracts/mod-generic/Cargo.toml:8]: sure, I'll change it to that

File: packages/croncat-sdk-tasks/Cargo.toml (hunk 3 above)
Comment (mikedotexe) [reply packages/croncat-sdk-tasks/Cargo.toml:8]: Not yet 😛 

Screen Shot 2023-02-04 at 7 28 50 PM

**Pull request merged**